include/macros.h

```diff
@@ -238,6 +238,9 @@ extern struct GraphicsContext* __gfxCtx;
 
 #endif /* OOT_DEBUG */
 
+#define gSPMATRIX_SET_NEW(pkt, gfxCtx, file, line) \
+    gSPMatrix(pkt, MATRIX_NEW(gfxCtx, file, line), G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW)
```

Comment:
```suggestion
    gSPMatrix(pkt, MATRIX_NEW(gfxCtx, file, line), G_MTX_MODELVIEW | G_MTX_LOAD | G_MTX_NOPUSH)
```

The order of the flags should be inverted. G_MTX_MODELVIEW/G_MTX_PROJECTION should be first since these determine which matrix stack is being modified. G_MTX_PUSH/G_MTX_NOPUSH should be last because these flags have no effect on the projection matrix stack and arguably could be completely omitted when G_MTX_PROJECTION is used.

Reply:
Sure, whatever. This order is just what the repo currently uses most but I can change it

Reply:
noticed doing so would also require a change to libgfxd